packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -59,12 +59,12 @@ function TabPanel(props) {
   );
 
   return (
-    <div className={className}>
+    <>
```

Comment:
I believe this classname was added a while back to support styled-component TabPanels. Is this not needed to do:
```
const myTab = styled(TabPanel)`
  *styles*
`;
```
?... and if not, should we remove the `className` parameter since it's no longer used?

Reply:
Yes. We definitely need this for the styled-components passthrough functionality Ken is referencing.

Reply:
yea...idk what was going on here....i'll remove the changes in this file.